ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -450,6 +451,14 @@ RnsShell::LayerInvalidateMask  RSkComponentTextInput::updateComponentProps(const
     caretHidden_ = true;
   }
   hasToSetFocus_ = forceUpdate && textInputProps.autoFocus ? true :false;
+
+/* Fetch OnSCreenKeyBoard Props*/
+  showSoftInputOnFocus_=textInputProps.traits.showSoftInputOnFocus;
+  oskLaunchConfig_.type=RSkToSdkOSKeyboardType(textInputProps.traits.keyboardType);
+  oskLaunchConfig_.theme=RSkToSdkOSKeyboardTheme(textInputProps.traits.keyboardAppearance);
+  oskLaunchConfig_.returnKeyLabel=RSkToSdkOSKReturnKeyType(textInputProps.traits.returnKeyType);
+  oskLaunchConfig_.enablesReturnKeyAutomatically=textInputProps.traits.enablesReturnKeyAutomatically;
+ 
```

Comment:
if these values are set during run time , should the OSK window be updated with these values or it is considered in next launch of OSK window

Reply:
If there are property update before the Texinput get Focus, changes will be updated and that get reflected. Otherwise it will not consider .Before OSK launch changes from App will be received by Texinput through property update and will be maintained.

As KB type to be defined by the App based on the need and there is no user provision to change the App type. So I don't see a use case where KB nature to be changed once it is launched.